tests/vse_sync_pp/analyzers/test_ppsdpll.py

```diff
@@ -21,54 +21,135 @@ class TestPhaseOffsetTimeErrorAnalyzer(TestCase, metaclass=AnalyzerTestBuilder):
     parser = 'dpll/phase-offset'
     expect = (
         (
```

Comment:
Could we make these have some form of named structure? I think it would be much easier if t was something like 
```python
expect = (
    {
      "expected": {
          "result": True,
          "stats": {...},
          "reason": None,
      "data": ( ...),
    },
)
```

Reply:
Easily, the 6-tuple was originally a 3?-tuple which was more readable.

Reply:
Yeah I can just about keep 3 things in my head :)

Reply:
Replaced each 6 tuple with a dict. The keys of the dict map directly to the variable names.